lib/xeroizer/models/organisation.rb

```diff
@@ -29,6 +29,8 @@ class Organisation < Base
       string    :registration_number
       string    :timezone
       datetime  :created_date_utc, :api_name => 'CreatedDateUTC'
+      string    :sales_tax_basis
```

Comment:
Should this validate that the param is one of http://developer.xero.com/documentation/api/types/#SalesTaxBasis  ? That'd be more helpful for end users